.github/workflows/staleness_check.yml

```diff
@@ -56,8 +56,7 @@ jobs:
           bash: >
             set -ex;
             if [[ -z $COMMIT_TRIGGERED_RUN || -z $MAIN_RUN ]]; then
-            bazel query 'attr(tags, "staleness_test", //...)' | xargs bazel test $BAZEL_FLAGS ||
-            echo "Please run ./regenerate_stale_files.sh to regenerate stale files";
+            bazel query 'attr(tags, "staleness_test", //...)' | xargs bazel test $BAZEL_FLAGS;
```

Comment:
The message seems useful. I didn't know the right command to run to fix this.

Would this work?

```
bazel query 'attr(tags, "staleness_test", //...)' | xargs bazel test $BAZEL_FLAGS || {
  echo "Please run ./regenerate_stale_files.sh to regenerate stale files";
  exit 1;
}
```

Reply:
I think so, but these tests are so fragile I wanted to keep it as simple as possible :(

Reply:
Yea that sgtm